### PR TITLE
React to aspnet/Razor#196.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- Razor `TagHelper`s now special case `enum` values so you don't need to provide the entire `enum` prefix.

### CODE REVIEW

**Comment by dougbu** [comment]
:shipit: assuming this is all that breaks for you locally.

File: samples/TagHelperSample.Web/Views/Home/Create.cshtml [hunk 1]
```diff
@@ -21,7 +21,7 @@
         @* - should helper remove the <div/> if list isn't generated? *@
         @* - (Html.ValidationSummary returns empty string despite non-empty message parameter) *@
         @* Acceptable values are: "None", "ModelOnly" and "All" *@
-        <div asp-validation-summary="ValidationSummary.All" style="color:blue" id="validation_day" class="form-group">
+        <div asp-validation-summary="@ValidationSummary.All" style="color:blue" id="validation_day" class="form-group">
```
Comment (Eilon) [inline samples/TagHelperSample.Web/Views/Home/Create.cshtml]: Any reason you didn't change this one to `All`?

Comment (NTaylorMullen) [inline samples/TagHelperSample.Web/Views/Home/Create.cshtml]: This is the sample site, there's another page in the site that has the non `@` variation. Wanted to showcase both ways.

Comment (Eilon) [inline samples/TagHelperSample.Web/Views/Home/Create.cshtml]: :+1:

**Pull request merged**